config/samples/bpfman.io_v1alpha1_bpfapplication.yaml

```diff
@@ -9,50 +9,113 @@ spec:
   nodeselector: {}
   bytecode:
     image:
-      url: quay.io/bpfman-bytecode/go-app-counter:latest
+      url: quay.io/bpfman-bytecode/app-test:latest
+      # imagepullpolicy: Always
   programs:
-    - type: Kprobe
+    - bpffunctionname: kprobe_test
+      type: Kprobe
       kprobe:
-        bpffunctionname: kprobe_counter
-        func_name: try_to_wake_up
-        offset: 0
-        retprobe: false
-    - type: Tracepoint
+        attach_points:
+          - func_name: try_to_wake_up
+            offset: 0
+            retprobe: false
+    - bpffunctionname: kretprobe_test
+      type: Kprobe
+      kprobe:
+        attach_points:
+          - func_name: try_to_wake_up
+            offset: 0
+            retprobe: true
+    - bpffunctionname: tracepoint_test
+      type: Tracepoint
       tracepoint:
-        bpffunctionname: tracepoint_kill_recorder
-        names:
-          - syscalls/sys_enter_kill
-    - type: TC
+        attach_points:
+          - name: syscalls/sys_enter_openat
+    - bpffunctionname: tc_pass_test
+      type: TC
       tc:
-        bpffunctionname: stats
-        interfaceselector:
-          primarynodeinterface: true
-        priority: 55
-        direction: ingress
-    - type: TCX
+        attach_points:
+          - interfaceselector:
```

Comment:
`interfaceSelector`. I'm not going to comment on this anymore.
We seriously need to take a pass through the API to fix the inconsistencies.
We can do that in this PR, or later